src/config/capabilityStatementResources.json

```diff
@@ -0,0 +1,267 @@
+      ],
+      "operation": [
+        {
+          "extension" : [{
+            "url" : "http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation",
+            "valueCode" : "SHALL"
+          }],
+          "name" : "package",
+          "definition" : "http://hl7.org/fhir/us/cqfmeasures/OperationDefinition/Library-package"
+        },
+        {
+          "extension" : [{
+            "url" : "http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation",
+            "valueCode" : "SHALL"
+          }],
+          "name" : "data-requirements",
+          "definition" : "http://hl7.org/fhir/us/cqfmeasures/OperationDefinition/Library-data-requirements"
+        }
+      ]
+    },
+    {
+      "type" : "Measure",
+      "profile" : "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/measure-cqfm",
+      "supportedProfile" : [
+        "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/computable-measure-cqfm",
+        "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/publishable-measure-cqfm",
+        "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cohort-measure-cqfm",
+        "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/proportion-measure-cqfm",
+        "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/ratio-measure-cqfm",
+        "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cv-measure-cqfm",
+        "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/composite-measure-cqfm"
+      ],
+      "interaction" : [
+        {
+          "extension" : [
+            {
+              "url" : "http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation",
+              "valueCode" : "SHALL"
+            }
+          ],
+          "code" : "read",
+          "documentation" : "Read allows clients to get the definitions and details of repository measures"
+        },
+        {
+          "extension" : [
+            {
+              "url" : "http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation",
+              "valueCode" : "SHALL"
+            }
+          ],
+          "code" : "search-type",
+          "documentation" : "Search allows clients to search for repository measures"
+        }
+      ],
+      "searchParam" : [
+        {
+          "extension" : [
+            {
+              "url" : "http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation",
+              "valueCode" : "SHALL"
+            }
+          ],
+          "name" : "url",
+          "definition" : "http://hl7.org/fhir/SearchParameter/Measure-url",
+          "type" : "uri"
+        },
+        {
+          "extension" : [
+            {
+              "url" : "http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation",
+              "valueCode" : "SHALL"
+            }
+          ],
+          "name" : "version",
+          "definition" : "http://hl7.org/fhir/SearchParameter/Measure-version",
+          "type" : "token"
```

Comment:
I believe that "version" is of type string. From the docs, it looks like token search params are typically used for parameters of code or identifier type. Since version is just of type string, I think this should just be string also?